### ISSUE TITLE
do not create BUFR from sample

### ISSUE THREAD

**Issue by tomkralidis** [World-Meteorological-Organization/wis2box#314]
For efficiency, in `wis2box/data/bufr4.py`, do not create new BUFR from samples (i.e. not from `codes_bufr_new_from_samples("BUFR4")`).

**Comment by david-i-berry** [comment]
I've been thinking on this. 

We can reportedly create the new BUFR by creating and saving a template and then by specifying this template in the call to `codes_bufr_new_from_samples`. This will not work with the workflow for a generic BUFR pipeline as it requires prior knowledge of the BUFR to be processed and the creation of a template before we start processing the data. One option around this may to be the creation of a template the first time a sequence is encountered but we then need to decide how to manage these templates. I think this may then be defeating the purpose of the generic BUFR pipeline.

**Comment by tomkralidis** [comment]
- Create additional samples based wis2box requirements
- ask ECMWF to add these samples into ecCodes samples, given they may be of use for others

**Comment by david-i-berry** [comment]
The following may be helpful but I haven't checked how to use these in the python library yet.

https://confluence.ecmwf.int/display/ECC/BUFR+templates

**Comment by tomkralidis** [comment]
@efucile to discuss with ECMWF to add additional templates to ecCodes.  @david-i-berry to provide suggestions, based on:

- review BUFR data from wis2box demos/data samples

**Comment by tomkralidis** [comment]
Needs:
- addition of templates in https://github.com/wmo-im/dim_eccodes_baseimage/blob/main/Dockerfile
- needs update in wis2box-management

Agree to lower priority.
